projects/community-payback-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/AppointmentsService.kt

```diff
@@ -249,9 +243,11 @@ class AppointmentsService(
         rowVersion = request.version.mostSignificantBits
     }
 
-    private fun Contact.update(request: AppointmentOutcomeRequest) = apply {
+    private fun Contact.update(request: AppointmentOutcomeRequest, contactOutcome: ContactOutcome?, staff: Staff) = apply {
         startTime = request.startTime
         endTime = request.endTime
+        this.staff = staff
+        this.contactOutcome = contactOutcome
         notes = notes?.let { it + System.lineSeparator() + System.lineSeparator() + request.notes }
```

Comment:
Not related to this PR, but does this work if there are no existing notes?  You might be better off with something like:
```
notes = listOfNotNull(notes, request.notes).joinToString("\n\n")
```